src/containers/Dev/Snapshots.tsx

```diff
@@ -244,7 +244,7 @@ const Snapshots: React.FC = () => {
                 showToast({
                   title: 'Download complete',
                   text: `${displayType === 'archives' ? 'ARCHIVE_' : ''}${filename}`,
-                  timeout: 30000,
+                  timeout: 0,
```

Comment:
Makes it stay permanently until clicked. I thought this was already set on this one.